repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
Yeah, I don't like it either, but I'm not sure how else to pass in the connection pool id without either manually creating the object (i.e. not injecting it) or doing some multi-threaded funkiness to ensure it's only set one time upfront (which could also make it a major version change to boot).  In theory (famous last words), this shouldn't cause an issues as the filters that are setting it are doing it before their configs are set to initialized, and the only time they'd ever set it after that would be if the config changes, and I believe a config change of this magnitude would result in Repose just restarting.